apis/launchers.md

```diff
@@ -61,48 +61,80 @@ As you can see, the method `set_progress` takes a `double` value type and is a r
 
 ## Actions
 
-Actions are specific functions your app can perform without already being open; think of them as alternate and more specific entry points into your app. Actions appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu.
+You can use any action defined in the `app` namespace as one of the entry points for your application. They appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu. Implementing actions is covered in-depth in [the actions section](actions).
 
-Adding actions to a `.desktop` file does not involve writing any code or using any external dependencies, though your app needs a way to distinguish between actions, e.g. with command line flags.
+### D-Bus activation
```

Comment:
Hm... I'm not sure I understand the scope of the separate PR.

We need DBus activation for launching actions. So more-less all the changes on the Launchers page (except perhaps the updated description) need to go together. And cross-linking between the Actions and Launchers pages only makes sense if Launchers actually uses actions.

The one thing we could publish separately is refactored code examples in the Actions page (the split between `activate` and `startup` methods). But that change alone does not offer new capabilities to developers so I personally don't see it as valuable on its own. I can still extract it into separate PR, if you'd like.

Reply:
Yeah it would reduce the size of this PR making it easier to re-review launching once we're ready for it and it would still be good advice for anyone writing actions in their apps right now. Plus I think the other style changes to the example .destkop file are still an improvement even without dbus activation and just good future proofing. Sorry for the confusion! I would like to get more of your work merged sooner than later if possible :)

Reply:
Ok, I've split the PR in two parts:

1.  #184, which does not depend on DBus activation, and so is independent from dock/#166
2. #185, for which we'll need to wait for the Dock patch to land

I tried to separate the 2 pull requests as cleanly as possible, so there should be no conflicts between them.